packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.controller.ts

```diff
@@ -148,25 +147,23 @@ const DEMO_FEATURES: Omit<
             enabled: true,
           },
           {
-            type: "experiment",
+            type: "experiment-ref",
```

Comment:
This feature is hard-coding the environment as production.  What happens if the org doesn't have a `production` environment? (we create one by default when they create an org, but they can always delete or rename it)

Might be safer to look in their org for all of the defined environments and create a copy of these same rules for every environment.

Reply:
Makes sense. I can build the rules programmatically based on the environments.

Reply:
This was addressed [here](https://github.com/growthbook/growthbook/pull/1499#issuecomment-1688972938).